src/devices/CharacterLcd/LcdInterface.cs

```diff
@@ -33,6 +33,12 @@ public abstract partial class LcdInterface : IDisposable
         /// <param name="values">Bytes to be send to the device</param>
         public abstract void SendData(ReadOnlySpan<byte> values);
 
+        /// <summary>
+        /// Sends data to the LCD device
+        /// </summary>
+        /// <param name="values">Char to be send to the device</param>
+        public abstract void SendData(ReadOnlySpan<char> values);
```

Comment:
I thought you were going to make this non-abstract and call into the byte version